hack/flakes/flake.nix

```diff
@@ -57,8 +57,8 @@
             ldflags = [ "-s" "-w" ];
           };
 
-          clusterawsadm = buildGo122Module rec {
-            name = "clusterawsadm";
+          clusterctl-aws = buildGo122Module rec {
```

Comment:
why do we always have to build these?

Reply:
They're not available from the nixpkgs in binary form. I did have a way to download these via nix from GH releases for packages but I hit problems so dropped back to building them.

Reply:
The thing is with building all these packages is that they take up a lot of resources on my CPU on first build. Not a blocker or anything I was just wondering.

Reply:
I'm thinking of different ways to do this (try to figure out nix language and provide an easy way to package github releases so no need to build; or use a nix remote cache) but in the meantime can we merge this and I can look at that separately?

Reply:
Gonna merge and look at this again later.